Mathlib/Analysis/Meromorphic/NormalForm.lean

```diff
@@ -171,7 +171,7 @@ theorem MeromorphicNFAt.order_eq_zero_iff (hf : MeromorphicNFAt f x) :
 /-- **Local identity theorem**: two meromorphic functions in normal form agree in a
 neighborhood iff they agree in a pointed neighborhood.
 
-See `ContinuousAt.eventuallyEq_nhdNE_iff_eventuallyEq_nhd` for the analogous
+See `ContinuousAt.eventuallyEq_nhd_iff_eventuallyEq_nhdNE` for the analogous
```

Comment:
Good catch! I wonder, though (pre-existing): shouldn't the lemma be named using `nbhd` instead of `nhd`?

Reply:
Yes. And also I'd like the continuousAt and the meromorphicAt to be phrased in the same way (currently, the sides are reversed). This will be for another PR, though.

Reply:
I'll be happy to approve such a mechanical clean-up PR